nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/main/java/org/apache/nifi/processors/splunk/QuerySplunkIndexingStatus.java

```diff
@@ -57,6 +58,7 @@
 @SeeAlso(PutSplunkHTTP.class)
 public class QuerySplunkIndexingStatus extends SplunkAPICall {
     private static final String ENDPOINT = "/services/collector/ack";
+    private static final String ACKCHECKED_ATTRIBUTE = "ack.checked";
```

Comment:
I would prefix this with "splunk." or something similar. This is because later processors will have not context about what kind of acknowledgement is this

Reply:
Thanks Bence, corrected in next commit